bundles/ch.elexis.kassenbuch/src/ch/elexis/buchhaltung/kassenbuch/BuchungsDialog.java

```diff
@@ -87,8 +91,16 @@ protected Control createDialogArea(Composite parent) {
 		text = new Text(ret, SWT.BORDER);
 		text.setLayoutData(SWTHelper.getFillGridData(1, true, 1, false));
 		if (act == null) {
-			last = KassenbuchEintrag.recalc();
-			liBeleg.setText(KassenbuchEintrag.nextNr(last));
+			CompletableFuture<KassenbuchEintrag> future = CompletableFuture.supplyAsync(KassenbuchEintrag::recalc,
+					executor);
```

Comment:
Bitte in `CompletableFuture.runAsync(KassenbuchEintrag::recalc);` ändern. Da der Rückgabewert nicht benötigt wird, da last anders ermittelt wird, wird kein `CompletableFuture` benötigt.